.github/workflows/migration.yml

```diff
@@ -40,6 +40,7 @@ jobs:
             apt install -y sbcl unzip libsqlite3-dev make curl gawk freetds-dev libzip-dev && \
             git clone https://github.com/dimitri/pgloader.git && \
             cd pgloader && \
+            git checkout $(git tag --list --sort=-v:refname "v*" | head -n1) && \
```

Comment:
1/5 Should we pin a tag instead of using the latest? That would make it easier to avoid issues with new releases.

Reply:
That would also work, I suppose it depends on the goal.

I interpreted the migration code as wanting to access the latest version, but since `master` branch is too fragile, the latest tag would be the compromise between bleeding edge and stability
The maximum stability would be achieved using the `apt` repository's version of pgloader, but I see this was explicitly changed in [this PR](https://github.com/mattermost/mattermost/pull/25465/files#diff-9e794052ffc837d6f258ea584b3fc112aa43c97751c5e2277b6470ae4457d8bbR34)

Reply:
Either way this is disabled now, this may be decided at fixing time